md/BonitaBPM_platform_setup.md

```diff
@@ -102,27 +109,69 @@ If you want to modify the configuration of an already initialized Bonita BPM Pla
 Note that the *Platform Setup tool* does not need the Bonita BPM Server to be running for the configuration to be updated. However, the server needs to be restarted for the changes to take effect.
```

Comment:
I would recommend to provide instruction to pull, push and then restart the server instead of stopping the server first.
Also I think info message can be turned into a warning and simplify it by having only "The server needs to be restarted for the configuration changes to take effect.". 
Info that setup tool only need database access can be added at the top of the page in "Platform setup tool overview" section.